components/31-molecules/partner-block/partner-block.twig

```diff
@@ -0,0 +1,79 @@
+<div class="partner-block box-no-icon {{ modifier }}">
```

Comment:
Is there no semantic tag for this?
Section maybe?

Reply:
Section within a div? Isn't that a bit unnecessary here?

Reply:
DIV doesn't really count, you can have as many wrappers as you like.

I think it's best to make clear that it's a distinct part of whatever semantic element it's used in (I hope footer).

@hstandaert ?

Reply:
Done.